backend/clickhouse/migrations/000010_log_count_daily_mv.up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE MATERIALIZED VIEW IF NOT EXISTS log_count_daily_mv
+ENGINE = SummingMergeTree
```

Comment:
Per our conversation earlier, why not AggregatingMergeTree?

Reply:
Documentation was sparse here but from what I could tell, SummingMergeTree is a special case supporting only `sum` aggregations and may be more performant for this use case